config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  root controller: :jobs, action: :index 
-  resources :hospitals, only: :index
+  root controller: :users, action: :index 
+    resources :users, only: [:create, :new]
+  devise_for :users
+  
+    resources :jobs, only: :index
```

Comment:
should these resources be indented? I think maybe indent should align with the scope line below